perf/auto-qual-test/deploy_latest_daily.sh

```diff
@@ -0,0 +1,9 @@
+git clone https://github.com/istio/tools
```

Comment:
Make sure you add the standard prolog to the top.
```
WD=$(dirname $0) .... 

```
etc otherwise you can run it from any other directory.

Reply:
Good catch.  Done.